spec/linter-codeclimate-spec.js

```diff
@@ -13,14 +14,15 @@ describe('The codeclimate provider for Linter', () => {
 
     waitsForPromise(() =>
       Promise.all([
-        atom.packages.activatePackage('language-ruby'),
         atom.packages.activatePackage('linter-codeclimate'),
       ]),
     );
   });
 
   it('works with a valid .codeclimate.yml file', () =>
-    waitsForPromise(() =>
+    waitsForPromise(
+      { timeout: 10000 },
```

Comment:
Atom sets 60 sec on CI, or 5 sec otherwise. This is exactly the same issue that was making the tests fail for me locally. You can read my comment at PR #65.